src/NzbDrone.Core/Download/TrackedDownloads/TrackedDownloadAlreadyImported.cs

```diff
@@ -7,12 +7,12 @@ namespace NzbDrone.Core.Download.TrackedDownloads
 {
     public interface ITrackedDownloadAlreadyImported
     {
-        bool IsImported(TrackedDownload trackedDownload, List<History.History> historyItems);
+        bool IsImported(TrackedDownload trackedDownload, List<Core.History.History> historyItems);
     }
 
     public class TrackedDownloadAlreadyImported : ITrackedDownloadAlreadyImported
```

Comment:
No longer used.

Reply:
Still used in `CompletedDownloadService`

Reply:
In that case, shouldn't this be updated to check DownloadHistory instead?